src/pudl/metadata/fields.py

```diff
@@ -48,6 +50,15 @@
         "unit": "USD",
     },
     "address_2": {"type": "string", "description": "Second line of the address."},
+    "adjacent_balancing_authority_code_eia": {
+        "type": "string",
+        "description": "EIA short code for the other adjacent balancing authority, with which interchange is occuring. Includes Canadian and Mexican BAs.",
+    },
+    "adjacent_region_code_eia": {
+        "type": "string",
+        "description": "EIA code for the other adjacent region, with which interchange is occuring.",
+        "constraints": {"enum": EIA_REGIONS},
+    },
```

Comment:
nit but i strongly prefer putting a qualifiers to a field name at or towards the end of a field name so the fields are easy to round up together. these would be `balancing_authority_code_adjacent_eia` and `region_code_adjacent_eia` bc we always put source at the very end